app/services/data_centers/merge_duplicates.rb

```diff
@@ -0,0 +1,85 @@
+module DataCenters
+  # WARNING: Run service with "run_update: yes" argument and you'll update, 
```

Comment:
I'd use `run_update: true`, seems to me closer to convention, but it's a detail.